include/mxnet/imperative.h

```diff
@@ -116,6 +116,7 @@ class Imperative {
     const std::unordered_set<uint32_t>& mutable_input_nodes() {
       return fwd_graph_.indexed_graph().mutable_input_nodes();
     }
+    nnvm::Graph &GetFullGraph(nnvm::Graph& g, const std::vector<nnvm::NodePtr> &ginputs); 
```

Comment:
I don't think this should be returning a reference, given that you return an rvalue that gets copied into another variable. I think returning the graph directly is the same.